src/pages/commits/CommitsChart.tsx

```diff
@@ -29,12 +29,20 @@ interface Props {
   versions?: Commits;
   hasTaskFilter?: boolean;
   hasError?: boolean;
+  isOpenChartToggle: boolean;
+  onToggleChartViewOptionsAccordion: ({
+    isVisible,
+  }: {
+    isVisible: boolean;
+  }) => void;
```

Comment:
`onToggleChart: (isVisible: boolean) => void`

Reply:
Accordion actually accepts an interface for the callback 
https://github.com/evergreen-ci/spruce/pull/1212/files#diff-65dca1bb07024427b8595b463e4e0c3e469f2c0f106e6c928a30fd41fb96acfdL14